rnote-ui/src/overlays.rs

```diff
@@ -260,7 +262,34 @@ impl RnOverlays {
                                 appwindow.handle_widget_flags(widget_flags, &canvas);
                             }
                         }
-                        PenStyle::Brush | PenStyle::Shaper | PenStyle::Eraser | PenStyle::Selector | PenStyle::Tools => {}
+                        PenStyle::Selector => {
+                            let mut widget_flags = engine.record(Instant::now());
+                            let selection_keys = engine.store.selection_keys_unordered();
+
+                            engine.store.restyle_strokes(&selection_keys, |style| {
+                                match style {
+                                    Style::Rough(rough_style) => rough_style.stroke_color = Some(stroke_color),
+                                    Style::Smooth(smooth_style) => smooth_style.stroke_color = Some(stroke_color),
+                                    Style::Textured(textured_style) => textured_style.stroke_color = Some(stroke_color),
+                                }
+                            }, |text_style| {
+                                text_style.color = stroke_color;
+                            });
+
+                            engine.store.update_geometry_for_strokes(&selection_keys);
+                            engine.store.regenerate_rendering_in_viewport_threaded(
```

Comment:
I am a little surprised the rendering is updated as shown in the clip, because here is missing:

```rust
store.set_rendering_dirty(&selection_keys);
```

To flag that the rendering is outdated